integrations/core/src/main/scala/bloop/integrations/BloopConfig.scala

```diff
@@ -0,0 +1,51 @@
+package bloop.integrations
+
+import java.io.{File, FileOutputStream}
+
+case class BloopConfig(
```

Comment:
If we really care about bincompat we should use protobuf :smile: However, bloop should not be used as a library, only as a server, so do you think this will be an issue?

Reply:
I believe it'd be nice to be able to offer a stable API for integration as a library. We can probably take care of that sometime in the future, when we get closer to a real release.

Reply:
Do you think this would be useful? Bloop is a builder server, it should run independently of any tool. What I think we should focus on is in the definition of a nice API :smile: